### ISSUE TITLE
Capital X after hyperlinks in 'word_document()'

### ISSUE THREAD

**Issue by kamapu** [rstudio/rmarkdown#2141]
I am having a strange issue in `word_document()`, namely after any hyperlink or at the end of a paragraph including a hyperlink, a capital X gets appended. For instance in this example:

```
---
title: "Document"
output:
  word_document:
    reference_docx: ms-template.docx
---

# Introduction

Here is a hyperlink: <https://github.com/>.

```

The template document is here: 
[ms-template.docx](https://github.com/rstudio/rmarkdown/files/6515192/ms-template.docx)

And here the result looks like here (Note the capital X at the end of line 3):
![Screenshot from 2021-05-20 12-41-05](https://user-images.githubusercontent.com/17233562/118965196-b1767580-b968-11eb-8490-b6f4eae094d9.png)

And here the details of my session:

```
R version 4.0.5 (2021-03-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.2 LTS

Locale:
  LC_CTYPE=en_US.UTF-8          LC_NUMERIC=C                 
  LC_TIME=de_DE.UTF-8           LC_COLLATE=en_US.UTF-8       
  LC_MONETARY=de_DE.UTF-8       LC_MESSAGES=en_US.UTF-8      
  LC_PAPER=de_DE.UTF-8          LC_NAME=de_DE.UTF-8          
  LC_ADDRESS=de_DE.UTF-8        LC_TELEPHONE=de_DE.UTF-8     
  LC_MEASUREMENT=de_DE.UTF-8    LC_IDENTIFICATION=de_DE.UTF-8

Package version:
  base64enc_0.1.3   digest_0.6.27     evaluate_0.14     glue_1.4.2       
  graphics_4.0.5    grDevices_4.0.5   highr_0.9         htmltools_0.5.1.1
  jsonlite_1.7.2    knitr_1.33        magrittr_2.0.1    markdown_1.1     
  methods_4.0.5     mime_0.10         rlang_0.4.11      rmarkdown_2.8.3  
  stats_4.0.5       stringi_1.6.2     stringr_1.4.0     tinytex_0.31     
  tools_4.0.5       utils_4.0.5       xfun_0.23         yaml_2.2.1       

Pandoc version: 2.11.3.1
```

May this be a bug in **rmarkdown**?

**Comment by kamapu** [comment]
Just an addition, I have tested it on `bookdown::word_document2()` and `officedown::rdocx_document()` and the result is the very same.

**Comment by cderv** [comment]
I can't reproduce with the default template 

````markdown
---
title: "Document"
output: word_document
---

# Introduction

Here is a hyperlink: <https://github.com/>.
````
![image](https://user-images.githubusercontent.com/6791940/118967042-81c86d00-b96a-11eb-96b8-45d8817334f1.png)

So it must comes from the template, or directly from Pandoc. 

Which version are you using ? `rmarkdown::pandoc_version()`

**Comment by cderv** [comment]
And I can't reproduce either using your template 

````markdown
---
title: "Document"
output: 
  word_document:
    reference_docx: ms-template.docx
---

# Introduction

Here is a hyperlink: <https://github.com/>.
````
![image](https://user-images.githubusercontent.com/6791940/118967389-ef749900-b96a-11eb-8d05-e4d1c95351a6.png)

I am using Pandoc 2.13, but I get the same with 2.11 or 2.7.3. 

Not sure why you would see this if you have nothing in your source Rmd after the link.

**Comment by kamapu** [comment]
Thanks @cderv for testing. I just updated **pandoc** to version 2.13 but the problem persists. My guess is that the problem is specific for Ubuntu (the operative system in my computer) or because the template is created with LibreOffice and not with Word.

I tested your first example using the default template and the X disappears, thus the problem may lie in the template.

Here a screenshot from my own document to show how bat it is:

![Screenshot from 2021-05-20 18-11-11](https://user-images.githubusercontent.com/17233562/119012954-cb2eb180-b996-11eb-8f70-b46ec32a9f00.png)

I'll try using an odt file instead.
